src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -161,3 +165,58 @@ def snapshot(self) -> Iterable[ClusterInfo]:
     def _try_fetch(self) -> Iterable[ClusterInfo]:
         for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
             yield ClusterInfo(*row)
+
+
+@dataclass
+class PolicyInfo:
+    policy_id: str
+    policy_name: str
+    success: int
+    failures: str
+    spark_version: str | None = None
+    policy_description: str | None = None
+    creator: str | None = None
+
+
+class PoliciesCrawler(CrawlerBase[PolicyInfo], CheckClusterMixin):
+    def __init__(self, ws: WorkspaceClient, sbe: SqlBackend, schema):
+        super().__init__(sbe, "hive_metastore", schema, "policies", PolicyInfo)
+        self._ws = ws
+
+    def _crawl(self) -> Iterable[PolicyInfo]:
+        all_policices = list(self._ws.cluster_policies.list())
+        return list(self._assess_policies(all_policices))
+
+    def _assess_policies(self, all_policices):
+        failures: list[str] = []
+        for policy in all_policices:
+            if policy.policy_id is None:
+                continue
+            failures.extend(self._check_cluster_policy(policy.policy_id, "policy"))
+            if "spark_version" not in json.loads(policy.definition):
+                spark_version = None
+            else:
+                spark_version = json.loads(policy.definition)["spark_version"]
```

Comment:
this is a bug - policy definition has a dictionary, we need to store a string.

Reply:
Storing as string now